pkg/volume/util/util.go

```diff
@@ -75,23 +75,41 @@ func SetReady(dir string) {
 // UnmountPath is a common unmount routine that unmounts the given path and
 // deletes the remaining directory if successful.
 func UnmountPath(mountPath string, mounter mount.Interface) error {
+	return UnmountMountPoint(mountPath, mounter, false) /* extensiveMountPointCheck = false */
```

Comment:
Super nit: 

I prefer 
```GO
UnmountMountPoint(mountPath, mounter, false /* extensiveMountPointCheck */)
```
since it's less verbose and doesn't require you to update the value and comment if you change the value.